src/.nuget/Microsoft.NETCore.ILAsm/runtime.Linux.Microsoft.NETCore.ILAsm.props

```diff
@@ -1,11 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <SkipPackageFileCheck>true</SkipPackageFileCheck>
-    <PackageTargetRuntime>linux-$(PackagePlatform)</PackageTargetRuntime>
-    <!-- only build for x64 -->
     <PackagePlatforms>x64;</PackagePlatforms>
+    <PackagePlatforms Condition="'$(RuntimeOS)' == 'ubuntu.14.04'">x64;arm;</PackagePlatforms>
```

Comment:
I don't think I can remove PackagePlatforms yet, because the filtering still exists.  That's a slightly more complex fix to remove PackagePlatforms.  I'll leave this as-is for now.